core/src/main/scala/cats/Parallel.scala

```diff
@@ -0,0 +1,100 @@
+package cats
+
+/**
+  * Some types that form a Monad, are also capable of forming an Applicative that supports parallel composition.
+  * The Parallel type class allows us to represent this relationship.
+  */
+trait Parallel[M[_], F[_]] {
+  /**
+    * The applicative instance for F[_]
+    */
+  def applicative: Applicative[F]
+
+  /**
+    * Natural Transformation from the sequential Monad M[_] to the parallel Applicative F[_].
```

Comment:
the other way? `from  the parallel Applicative F[_] to the sequential Monad M[_]`